contrib/time_series_expectations/assets/generate_test_time_series_data.py

```diff
@@ -0,0 +1,206 @@
+"""Generate test data for time series. Includes both CSVs and PNGs"""
+
+import os.path
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from great_expectations.data_context.util import file_relative_path
+from time_series_expectations.generator import generate_daily_time_series_df
+
+plt.rcParams["figure.figsize"] = (20, 3)
+
+
+def def_generate_time_series_data_and_plot(
```

Comment:
nit, looks like an extraneous `def_`?
```suggestion
def generate_time_series_data_and_plot(
```

Reply:
Wow. That's what I get for coding late at night.